src/components/MarkdownProvider/index.tsx

```diff
@@ -65,6 +66,7 @@ export const MarkdownProvider: React.FC = ({ children }) => (
         BestPractice,
         Markdown,
         OverviewLinks,
+        Link,
```

Comment:
I don't think we want to import `Link` here for use within MDX pages. Should just be able to use `<a>` tags throughout.

Reply:
Do you think we'll want to do that across the board? Seems a shame to not leverage the Gatsby link to prevent full page re-renders every time (at least in some capacity, though I would vote to use it predominantly cause it's so slick in use).

Reply:
Or is there a future approach where we could leverage that functionality in the MDX if it's an internal page link?

Reply:
I'm going to let @hzhu and @austin94 weigh-in here. Not sure what the best approach is. Using `<a>` feels simplest to me. But whatever we decide, it needs to be consistent across the board.

Reply:
For the interim, turned this into an `<a>` tag so as not to block.

Reply:
If we are unsure of URL destination let's leave it as `<a>` for now

Reply:
@rossmoody I think the usages you had were okay. For consistency, we may want a separate PR post V1 to move all local links to use Gatsby's `Link` component.

Source: [Replace `a` tags with the `Link` tag for local links](https://www.gatsbyjs.org/docs/gatsby-link/#replace-a-tags-with-the-link-tag-for-local-links)

Reply:
@hzhu in that case, the change should be made in the central `MarkdownProvider` and then expressed as markdown (within `Markdown` tags if needed).